pages/_app.tsx

```diff
@@ -114,8 +117,10 @@ export default function PlanetWeb({ Component, pageProps, err }: any) {
     storeConfig();
   }, []);
   React.useEffect(() => {
-    i18next.initPromise.then(() => setInitialized(true));
-  }, []);
+    if (i18n && i18n.isInitialized) {
+      setInitialized(true);
```

Comment:
Maybe you could add some code here to prevent adding it to every single page?
```typescript
        if (localStorage.getItem('i18nextLng') !== null) {
          const languageFromLocalStorage: any = localStorage.getItem('i18nextLng');
          i18n.changeLanguage(languageFromLocalStorage);
        }
```
Or you can try to find out how this code was meant to work and if you just can adapt it you also work with this pull request now: https://github.com/Plant-for-the-Planet-org/planet-webapp/blob/22cf2fd36c8da643df3bdca078a7a8f435bf207c/src/features/common/Layout/QueryParamsContext.tsx#L65-L70